adafruit_circuitplayground/express.py

```diff
@@ -50,6 +52,80 @@ def __init__(self):
         self.sample = None
         self.sine_wave = None
 
+        # Define acceleration:
+        self._i2c = busio.I2C(board.ACCELEROMETER_SCL, board.ACCELEROMETER_SDA)
+        self._lis3dh = adafruit_lis3dh.LIS3DH_I2C(self._i2c, address=0x19)
+        self._lis3dh.range = adafruit_lis3dh.RANGE_8_G
+        self._lis3dh._write_register_byte(adafruit_lis3dh.REG_CTRL5, 0b01001000)
+        self._lis3dh._write_register_byte(0x2E, 0b10000000)
+
+    def acceleration_update(self):
+        """``acceleration_update`` updates the acceleration data from the x, y
+        and z axes. It is only needed once to update all three.
+
+            .. image :: /_static/accelerometer.jpg
+
+            .. code-block:: python
+
+              from adafruit_circuitplayground.express import cpx
+
+              while True:
+                  cpx.acceleration_update()
+                  print(cpx.acceleration_x, cpx.acceleration_y, cpx.acceleration_z, sep=", ")
+        """
+        self._x, self._y, self._z = self._lis3dh.acceleration
+
+    @property
+    def acceleration_x(self):
```

Comment:
Why do you have x, y and z as separate properties? I was expecting a single `acceleration` property that just returns `self._lis3dh.acceleration`.

Reply:
It was necessary to use separate properties to make them work in the code as `acceleration_x`, `acceleration_y` and `acceleration_z`. Otherwise we're typing `cpx.x`, `cpx.y`, and `cpx.z` which I thought could be a little unclear. As well, since including something that `= self._lisd3h.acceleration` is required in whatever loop you have in your code if you want x, y and z to continue updating, I thought being able to call the update property separately and having them all start with `acceleration` made it more obvious that they're all related.

I can change it if you'd rather it just be the single property.

Reply:
I was thinking it'd be:

`x, y, z = cpx.acceleration`